[2025-03-17T01:37:11.976+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T01:37:12.002+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_minio_dag.check_minio_connection manual__2025-03-17T01:37:10.986488+00:00 [queued]>
[2025-03-17T01:37:12.012+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_minio_dag.check_minio_connection manual__2025-03-17T01:37:10.986488+00:00 [queued]>
[2025-03-17T01:37:12.014+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-17T01:37:12.032+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): check_minio_connection> on 2025-03-17 01:37:10.986488+00:00
[2025-03-17T01:37:12.042+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_minio_dag', 'check_minio_connection', 'manual__2025-03-17T01:37:10.986488+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmpvogddc3n']
[2025-03-17T01:37:12.046+0000] {standard_task_runner.py:105} INFO - Job 19: Subtask check_minio_connection
[2025-03-17T01:37:12.047+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=374) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-17T01:37:12.048+0000] {standard_task_runner.py:72} INFO - Started process 375 to run task
[2025-03-17T01:37:12.122+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_minio_dag.check_minio_connection manual__2025-03-17T01:37:10.986488+00:00 [running]> on host 643acaa02cac
[2025-03-17T01:37:12.242+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_minio_dag' AIRFLOW_CTX_TASK_ID='check_minio_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-03-17T01:37:10.986488+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-17T01:37:10.986488+00:00'
[2025-03-17T01:37:12.243+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-17T01:37:12.412+0000] {logging_mixin.py:190} INFO - Conexi√≥n a MinIO exitosa!
[2025-03-17T01:37:12.413+0000] {logging_mixin.py:190} INFO - Buckets: []
[2025-03-17T01:37:12.414+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-17T01:37:12.429+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-17T01:37:12.430+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_minio_dag, task_id=check_minio_connection, run_id=manual__2025-03-17T01:37:10.986488+00:00, execution_date=20250317T013710, start_date=20250317T013712, end_date=20250317T013712
[2025-03-17T01:37:12.507+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T01:37:12.546+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-17T01:37:12.554+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
